packages/@glimmer/opcode-compiler/lib/opcode-builder.ts

```diff
@@ -747,22 +750,29 @@ export abstract class OpcodeBuilder<Locator = Opaque> extends StdOpcodeBuilder {
     this.load(Register.s0);
   }
 
-  invokeStaticComponent(
-    capabilities: ComponentCapabilities,
-    layout: CompilableProgram,
-    attrs: Option<CompilableBlock>,
-    params: Option<WireFormat.Core.Params>,
-    hash: WireFormat.Core.Hash,
-    synthetic: boolean,
-    block: Option<CompilableBlock>,
-    inverse: Option<CompilableBlock> = null
-  ) {
+  invokeStaticComponent({
```

Comment:
I believe this makes it a breaking change, we should label the PR (and probably start a list of things for tracking in Ember update)...

Reply:
Yep!

Reply:
OK, I added `BREAKING` to the PR title, and started a bulleted list for the things that we know are breaking. At some point, we should review the diff and fill in more known scenarios...